templates/advanced/test/unit/services/example.test.js

```diff
@@ -0,0 +1,47 @@
+const { expect } = require('chai');
+
+const ExampleService = require('../../../src/services/example');
+
+describe('ExampleService', () => {
+  describe('General', () => {
```

Comment:
Maybe add a `beforeEach` to the top which creates an instance of their `ExampleService` so that they can re-use it in each test

Reply:
Implemented.